### PR TITLE
mumble,murmur: 1.2.19 -> 1.3.0

### PR DESCRIPTION
**Pull request by greizgh**
###### Motivation for this change

[Mumble v1.3.0](https://www.mumble.info/blog/mumble-1.3.0-release-announcement/) was officially released few weeks ago. This PR aims to update the package for both client (mumble) and server (murmur).
This also updates to the qt5's makeDerivation (see #67270).
I removed the now useless murmur_rc and mumble_rc packages, but I don't know how you feel about it?

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [x] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

###### Notify maintainers

cc @

### CODE REVIEW

File: pkgs/top-level/aliases.nix [hunk 1]
```diff
@@ -437,10 +437,6 @@ mapAliases ({
     gst-plugins-ugly = pkgs.gst-plugins-ugly;
   };
 
-  # added 2019-08-01
-  mumble_git = pkgs.mumble_rc;
-  murmur_git = pkgs.murmur_rc;
```
Comment (Infinisil) [inline pkgs/top-level/aliases.nix]: I think these should stay but point to `pkgs.{mumble,murmur}` instead. This then won't break people's configs.

**Review by infinisil — CHANGES_REQUESTED** [review]

File: pkgs/top-level/aliases.nix (hunk 1 above)
Comment (greizgh) [reply pkgs/top-level/aliases.nix]: Thank you for your feedback, aliases have been re-added.

**Comment by Infinisil** [comment]
Nice, tested it and it's working well. Can you squash the commits into one?

**Comment by greizgh** [comment]
Sure! Here you go, squashed and rebased.

**Review by infinisil — APPROVED** [review]

**Comment by ajs124** [comment]
This (seems to) break(s) mumble.override. As in, building `mumble.override { pulseSupport = true; }` doesn't work anymore.

**Comment by Infinisil** [comment]
@ajs124 Thanks for the report, this was easy to miss, I've opened https://github.com/NixOS/nixpkgs/pull/70685 to fix this

**Pull request merged**